Source/Private/PerfGrapherCommandlet.cpp

```diff
@@ -2,15 +2,17 @@
 
 #include "Chaos/AABB.h"
```

Comment:
that specific include is not needed (it's a resharper bug)

Reply:
Done